### DISCUSSION TITLE
Time Picker

### DISCUSSION THREAD

**Discussion by This-Marinov-guy** [gpbl/react-day-picker#2206] [Ideas]
Hello,

I really want to donate to the developers - but in order to use this package, I need to be sure that there is also time picker (only hours and minutes). If there is or there is plan to be added, give me a link and I am donating :)

**Comment by gpbl** [comment]
Thanks for the suggestion! DayPicker won't handle hours or minutes. 

To implement a time picker you can use two input fields, one for setting the hours, the other for setting the minutes. You would then store the value from these two fields in your component's state. 

When you need to use the date picked in DayPicker, make sure to set its hours and minutes with the values in the component state.
